source/vocab/details.ttl

```diff
@@ -946,6 +950,17 @@
     rdfs:subPropertyOf :place ;
     rdfs:range :Country .
 
+:datePublished a owl:DatatypeProperty;
+    :category :shorthand;
+    owl:equivalentProperty sdo:datePublished;
+    rdfs:subPropertyOf dc:issued;
+    owl:propertyChainAxiom ( :publication :date ) .
+
+:publisher a owl:ObjectProperty;
+    :category :shorthand;
+    owl:equivalentProperty dc:publisher, sdo:publisher;
+    owl:propertyChainAxiom ( :publication :agent ) .
```

Comment:
Maybe add a note that we also atm have ~600k linked publisher roles in contribution?

Reply:
*Maybe*, but only if our stated intent is to accept *and formalize* that odd (IMHO) way of describing publication (as being a kind of contribution to the work?!), and that the only place we'll attempt to address and normalize that is to be here, in this simple shorthand property.

And if that is to be the solution, what would the inverse reasoning be from e.g. a piece of DC or Schema.org description for a `:publisher` relation back to BF/KBV? Only a `:publication / :agent` chain? Or *also* a `:contribution / :agent` (both with `:role :publisher`)?